standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
 
 Finally, it should be noted that the "links" in this chain are formed by individual HTTPS request operations (against the various bundle endpoint servers in the chain), and these operations are likely occurring at different times. SPIFFE bundle endpoint clients should be logging these HTTPS request operations, and administrators should take care to preserve these logs for future forensic analysis if necessary.
 
+### 7.7 Processing claims asserted by foreign trust domains
+
+SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains. By an "authentic" claim, we mean specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim. This is not the same thing as saying that the claim is _true_, or that it is _correctly interpreted_.
+
+Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains. The recommended posture is not to process foreign claims that have not been evaluated accoring to the framework described here (i.e. default deny), then assert exceptions for specific claims from specific trust domains to be allowed if they pass muster.
+
+It bears repeating that implementations should never process any claims from SVIDs that fail authentication (except in a forensic context).
+
+#### 7.7.1 SVID Claims
+
+For the purpose of this discussion, a *claim* is an assertion about some attribute of the subject of an SVID, or about the SVID itself. The canonical SPIFFE claim is the SPIFFE ID, which is a claim about the subject of the SVID. An example of a claim about the SVID itself is the expiration time.
+
+SVID formats (to date including X.509 and JWT) afford flexibility to encode additional claims, including claims that have not been evaluated by any standards body.
+
+In considering how to process claims made by foreign trust domains, it is helpful to consider three categories of claims:
+
+  1. Claims required by and/or defined by the SPIFFE specifications for the SVID type
+  2. Claims that are standardized by the relevant standards body for the document format (i.e. ITU-T for X.509, IETF for JWT)
+  3. Non-standardized claims
+
+Claims in the first category are well-defined in their meaning, and in many cases implementations are required to process them to adhere to SPIFFE specifications.
+
+Claims in the second category have standardized meaning, but operators have some choice whether to and how to process them.
+
+Claims in the third category are not standarized, so operators need to consider them on a case-by-case basis, possibly including navigating different meanings by different trust domains. Needless to say, this category should be approached with the highest degree of caution.
```

Comment:
```suggestion
Claims in the third category are (by definition) not standardized, so operators need to consider them on a case-by-case basis, possibly including navigating different meanings by different trust domains. This category should be approached with the highest degree of caution.
```

Reply:
"different meanings by different trust domains" is a terrific point to make. I hope that there is an example or more text about this below.